core/src/com/raccoon/mygame/models/Guard.java

```diff
@@ -1,5 +1,67 @@
 package com.raccoon.mygame.models;
 
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.math.*;
+import com.badlogic.gdx.graphics.*;
+import com.raccoon.mygame.view.GameCanvas;
+
 public class Guard {
+    private Vector2 position;
+
+    private float width;
+    private float height;
+
+    private Vector2 velocity;
+
+    private Texture patrolTexture;
+
+    Guard(float x, float y, float width, float height, Texture texture){
+        position = new Vector2(0.0f, 0.0f);
```

Comment:
shouldn't it be position = new Vector2(x,y)?

Reply:
mmm true just changed and commited